aerogear/graphback
# fix(cli): restore generate command

The commands directory was removed in a previous commit, which means the CLI has not been working for a while.

```bash
Error: ENOENT: no such file or directory, scandir '/code/aerogear/graphback/packages/graphback-cli/dist/commands'
```

This restores the `generate` command, which is required for generation of client documents.

## Verification

1. Add a `.graphqlrc` file to your project.

```yaml
schema: './src/schema.graphql'
documents: './client/src/graphql/**/*.graphql'
extensions:
  graphback:
    model: ./model/datamodel.graphql
    plugins:
      graphback-client:
        # format: 'graphql'
        outputFile: ./client/src/graphql/graphback.graphql
```

2. Run `yarn graphback generate`. 
3. The command should run successfully.

## Tasks

- [x] Fix issue
- [x] Add integration test

## Notes

I tried to run the CLI through yarn in the integration tests. This worked fine locally but I had issues with CircleCI, so I changed it to call it with Node.js directly: https://github.com/aerogear/graphback/pull/1693/commits/349272c867f6241808cdfac6e5bf9d4e37eeb38a
